### PR TITLE
New version: IBM.SemeruRuntimeOpenEdition.JDK.17 version 17.0.9+9

### PR DESCRIPTION
**Pull request by kazssym**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/130429)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @kazssym,

> Current manifest version 17.0.7.7 has 4 parameters, which is greater than this PR's version number 17.0.9+9's 3 parameters.

Is this intentional?

(Automated response - build 485)

File: manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: IBM.SemeruRuntimeOpenEdition.JDK.17
+PackageVersion: 17.0.9+9
```
Comment (Trenly) [inline manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.installer.yaml:5]:
```suggestion
PackageVersion: 17.0.9.9
```
Should match control panel exactly. Version folder will need to be renamed after making this change

File: manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: IBM.SemeruRuntimeOpenEdition.JDK.17
+PackageVersion: 17.0.9+9
```
Comment (Trenly) [inline manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 17.0.9.9
```
Should match control panel exactly. Version folder will need to be renamed after making this change

File: manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: IBM.SemeruRuntimeOpenEdition.JDK.17
+PackageVersion: 17.0.9+9
```
Comment (Trenly) [inline manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.yaml:5]:
```suggestion
PackageVersion: 17.0.9.9
```
Should match control panel exactly. Version folder will need to be renamed after making this change

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**